### PR TITLE
Adding quick installation script for installing MXNet with Python on Amazon Linux

### PR DESCRIPTION
**Pull request by sandeep-krishnamurthy**
* Added bash script that installs mxnet with python on Amazon Linux machines.
* Added corresponding documentations in setup docs page.

### CODE REVIEW

File: setup-utils/install-mxnet-amz-linux.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+cmake -D BUILD_opencv_gpu=OFF -D WITH_EIGEN=ON -D WITH_TBB=ON -D WITH_CUDA=OFF -D WITH_1394=OFF -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=/usr/local ..
+make PREFIX=/usr/local install
+
+# Export env variables for pkg config
+export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH
+
+# Install MXNet Core without CUDA
+cd ~/MXNet/mxnet
+cp make/config.mk .
+echo "USE_CUDA=0" >>config.mk
```
Comment (piiswrong) [inline setup-utils/install-mxnet-amz-linux.sh]: no cuda?
use mkldnn?

File: setup-utils/install-mxnet-amz-linux.sh [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+echo "ADD_CFLAGS += -I/usr/include/openblas" >>config.mk
+echo "ADD_LDFLAGS += -lopencv_core -lopencv_imgproc -lopencv_imgcodecs" >>config.mk
+make -j$(nproc)
+
+# Install MXNet Python package
+cd python
+python setup.py install
+
+# Add MXNet path to ~/.bashrc file
+echo "export PYTHONPATH=~/MXNet/mxnet/python:$PYTHONPATH" >> ~/.bashrc
```
Comment (piiswrong) [inline setup-utils/install-mxnet-amz-linux.sh]: can we install things under ~/mxnet directly?

File: setup-utils/install-mxnet-amz-linux.sh (hunk 1 above)
Comment (sandeep-krishnamurthy) [inline setup-utils/install-mxnet-amz-linux.sh]: This is plain bare metal installation. We haven't added support in script to install cuda and mkldnn dependencies. We will upgrade the script later.

File: setup-utils/install-mxnet-amz-linux.sh (hunk 2 above)
Comment (sandeep-krishnamurthy) [inline setup-utils/install-mxnet-amz-linux.sh]: Updated.

File: docs/get_started/setup.md [hunk 3]
```diff
@@ -80,12 +91,13 @@ sudo apt-get -y install git
 ```
 
 Clone the MXNet source code repository to your computer, run the installation script, and refresh the environment variables. In addition to installing MXNet, the script installs all MXNet dependencies: ```Numpy```, ```LibBLAS``` and ```OpenCV```.
+It takes around 5 minutes to complete the installation.
 
 ```bash
 # Clone mxnet repository. In terminal, run the commands WITHOUT "sudo"
 git clone https://github.com/dmlc/mxnet.git ~/MXNet/mxnet --recursive
```
Comment (piiswrong) [inline docs/get_started/setup.md]: also change this?

**Comment by sandeep-krishnamurthy** [comment]
Updated ubuntu quick installation script also to install mxnet in ~/mxnet rather than ~/MXNet/mxnet.

**Pull request merged**